tests/e2e/default/more-options-menu/online-user-permissions-disabled.wdio-spec.js

```diff
@@ -0,0 +1,180 @@
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionVisible('export', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionEnabled('edit', 'contacts')).to.be.true;
+      expect(await commonPage.isOptionEnabled('delete', 'contacts')).to.be.true;
+    });
+  
+    it('- options enabled when report selected', async () => {
+      await commonPage.goToReports();
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;    
+      (await reportPage.firstReport()).click();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionVisible('export', 'reports')).to.be.false;
+      expect(await commonPage.isOptionVisible('edit', 'reports')).to.be.false; //not xml report
+      expect(await commonPage.isOptionEnabled('delete', 'reports')).to.be.true;     
+    }); 
+
+    it('- Message tab', async () => {
+      await commonPage.goToMessages();
+      await commonPage.waitForLoaderToDisappear();
+      expect(await commonPage.isMessagesListPresent()).to.be.true;
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;    
+    }); 
+  });
+
+  describe('- DELETE permissions disabled', async () => {
+    before(async () => {
+      await updatePermissions(onlineUser.roles, [], ['can_delete_contacts', 'can_delete_reports']);
+      await commonPage.closeReloadModal();
+    });
+
+    after(async () => {
+      await utils.revertSettings(true);
+    });
+
+    it(' - Contact Tab : contact selected', async () => {
+      await commonPage.goToPeople();
+      await contactPage.selectLHSRowByText(contact.name);
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionVisible('delete', 'contacts')).to.be.false;
+    });
+  
+    it('- options enabled when report selected', async () => {
+      await commonPage.goToReports();
+      (await reportPage.firstReport()).click();
+      await commonPage.openMoreOptionsMenu(); 
+      expect(await commonPage.isOptionVisible('delete', 'reports')).to.be.false;     
+    });  
+  });
+
+  describe('- EDIT permissions disabled', async () => {
+    before(async () => {
+      await updatePermissions(onlineUser.roles, [], ['can_edit']);
+      await commonPage.closeReloadModal();
+    });
+    
+    after(async () => {
+      await utils.revertSettings(true);
+    });
+    
+    it(' - Contact Tab : contact selected', async () => {
+      await commonPage.goToPeople();
+      await contactPage.selectLHSRowByText(contact.name);
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionVisible('edit', 'contacts')).to.be.false;
+    });
+  
+    it('- options enabled when report selected', async () => {
+      await commonPage.goToReports();
+      (await reportPage.firstReport()).click();
+      await commonPage.openMoreOptionsMenu(); 
+      expect(await commonPage.isOptionVisible('edit', 'reports')).to.be.false;     
+    });  
+  });
+});
```

Comment:
If this is about online user, why is it in a separate file than `online-user.wdio-spec.js`?